src/containers/GetAnArticle.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import AnArticle from '../components/AnArticle';
+import Comments from '../components/Comments';
+import {
+  fetchAnArticle,
+  fetchArticleComments,
+} from '../store/actions/commentsActions';
+
+export class GetAnArticle extends Component {
+  constructor(props) {
+    super(props);
+    this.slug = '';
```

Comment:
@Etomovich, it is better to use state instead of `this.slug`. This is not consistent in the app and will make maintenance complex and expensive.

Reply:
Got that and am handling it